__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("200: response array should contain all of the data from topics.", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics.length).toBe(3);
+        topics.forEach(({ description, slug }) => {
+          expect(typeof description).toBe("string");
+          expect(typeof slug).toBe("string");
+        });
+      });
+  });
+  test("if a non-existent url has been input, return the error message of Route not found", () => {
```

Comment:
Great test but please have this in its own describe block as it does not relate to any specific endpoint